cosmos-auth/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cosmos-auth",
   "description": "Node.js app exposing Cosmos Auth API",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Shouldn't it be 0.1.0-next or 0.1.0_SNAPSHOT instaed of a concret number such as "0.1.1"? At least that is the criteria used in other FIWARE components...